docs/guides/render-static-scenes.mdx

```diff
@@ -309,7 +309,7 @@ Open `scene-image.jpg` to view your scene
 
 ### In the Viewer component
 
-To view the scene, create a stream key [described here](./authentication.mdx#in-the-viewer-sdk) and pass it to the `viewer.load` method. When loading scenes with the Viewer component, Vertex creates scene views as described above.
+To view the scene, create a stream key [described here](./authentication.mdx#viewer-component) and pass it to the `viewer.load` method. When loading scenes with the Viewer component, Vertex creates scene views as described above.
```

Comment:
✔️ https://stackoverflow.com/questions/2822089/how-to-link-to-part-of-the-same-document-in-markdown#comment58139889_16426829